scripts/postCommentOnPR.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+const GithubApi = require('github');
+
+// These env variables all need to be defined.
+const { GHE_TOKEN, TRAVIS_REPO_SLUG, TRAVIS_PULL_REQUEST } = process.env;
+
+// When invoked using `node postCommentOnPR.js "some message", the string we
+// want is third in the list.
+const body = process.argv[2];
+
+const github = new GithubApi();
+
+github.authenticate({
+  type: 'token',
+  token: GHE_TOKEN
+});
+
+const [owner, repo] = TRAVIS_REPO_SLUG.split('/');
+
+// Post comment on PR. We're not going to fail or pass any build, we're just
```

Comment:
This seems like a reasonable way to notify users of the status, but I feel that it would be an improvement to integrate with rest of the PR status checks.  Administrator's can then easily configure whether it should be required or not.

In the current implementation, there is potential to be a lot of Happo review comments as PR gets updated.  With status checks, everything would be nice and consolidated to the status check section.  You could also include the cute little Happo logo, the overall status, and hide the big ugly url behind the details link.

![screen shot 2017-11-15 at 8 56 49 am](https://user-images.githubusercontent.com/202773/32845811-fb056950-c9e2-11e7-97f3-93d6c661a038.png)

Reply:
Agreed. It’s on my todo-list for the future. In the meantime, we can make it only post the comment if there is a diff. This is how we do in a few other projects and it’s an easy fix. I’d suggest we start with this and address when it’s too noisy. Sound good?

Reply:
Sure - I was just providing some of that constructive feedback that you requested. 😄

Reply:
Much appreciated! 😊

Reply:
Followup question related to this subject...

When the VR images do not match - something addded/removed/changed, it looks like this causes a travis build failure for the `TEST_SUITE=happo` portion of the build.

<img width="1019" alt="screen shot 2017-11-16 at 6 33 32 am" src="https://user-images.githubusercontent.com/202773/32894015-bf5857bc-ca98-11e7-9ce4-00c74eba5bdf.png">

This however causes a failure for the entire Travis status check.

<img width="736" alt="screen shot 2017-11-16 at 6 33 09 am" src="https://user-images.githubusercontent.com/202773/32894054-e147bb38-ca98-11e7-9d26-b117fd30abe8.png">

Since we have Travis marked as a required check, this means that the PR will be blocked.  Does this mean that we'll need to make it an optional status check until Happo supports its own status check?